src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/AccessibleObjects/ToolStripDropDownButton.ToolStripDropDownButtonAccessibleObjectTests.cs

```diff
@@ -92,4 +93,33 @@ public void ToolStripDropDownButtonAccessibleObject_FragmentNavigate_Child_Retur
         Assert.Equal(expected, accessibleObject.FragmentNavigate(NavigateDirection.NavigateDirection_FirstChild));
         Assert.Equal(expected, accessibleObject.FragmentNavigate(NavigateDirection.NavigateDirection_LastChild));
     }
+
+    private ToolStripDropDownButton CreateToolStripDropDownButton(string text = null, Bitmap image = null, EventHandler onClick = null, string name = null, ToolStripItem[] dropDownItems = null)
+    {
+        ToolStripDropDownButton toolStripDropDownButton = new(text, image, dropDownItems);
+        toolStripDropDownButton.Click += onClick;
+        toolStripDropDownButton.Name = name;
+
+        return toolStripDropDownButton;
+    }
+
+    [WinFormsFact]
+    public void ToolStripDropDownButton_ConstructorParameters_ShouldInitializeCorrectly()
+    {
+        string text = "Test text";
+        Bitmap image = new(10, 10);
+        bool wasClicked = false;
+        EventHandler onClick = (sender, e) => { wasClicked = true; };
+        string name = "Test name";
+        var dropDownItems = new ToolStripItem[] { new ToolStripMenuItem("Test item") };
```

Comment:
I believe we can initialize this one like so
```suggestion
        ToolStripItem[] dropDownItems = [new ToolStripMenuItem("Test item")];
```